### PR TITLE
[panels] update attribute-template.json

### LINKED ISSUE #204: index_pattern is not up to date in the templates
I was facing problems when I tried to visualize the QM using Prosoul.

But the result of the dashboard was empty.

The whole procedure can be found at https://github.com/vchrombie/chaoss-microtasks/issues/11#issuecomment-604919542. 

With the help of @valeriocos, I realized that the dashboards are not updated automatically. The index_pattern isn't up to date in the [attribute-template.json](https://github.com/Bitergia/prosoul/blob/master/django-prosoul/prosoul/panels/templates/attribute-template.json).

### PR DESCRIPTION
**Pull request by vchrombie**
This PR updates the panels template for creating a dashboard during visualization.

Fixes #204

### CODE REVIEW

**Comment by vchrombie** [comment]
Hi @valeriocos 
I completed the work as stated in the #204. I exported the updated dashboard using Kidash.

I have a few doubts though. Please review this PR when you have time.

File: django-prosoul/prosoul/panels/templates/attribute-template.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
 {
     "dashboard": {
-        "id": "AWG5dj8SETw-8I6Z4dX-",
+        "id": "activity_docker-smells",
```
Comment (vchrombie) [inline django-prosoul/prosoul/panels/templates/attribute-template.json]: I am just curious to know if I have to revert this change back. :thinking:

File: django-prosoul/prosoul/panels/templates/attribute-template.json [hunk 2]
```diff
@@ -26,7 +27,7 @@
...
...ues\":true},{\"name\":\"metric_es_compute\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"metric_es_value\",\"type\":\"number\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"metric_es_value_weighted\",\"type\":\"number\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"metric_id\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"metric_name\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"metric_type\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"project\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true},{\"name\":\"uuid\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true}]",
```
Comment (vchrombie) [inline django-prosoul/prosoul/panels/templates/attribute-template.json]: I understand that this is the change we are aiming to solve in the issue #204. 
Do I have to revert back the other changes?

Please correct me if I am wrong.

File: django-prosoul/prosoul/panels/templates/attribute-template.json [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
                 "display": "Off",
                 "pause": false,
                 "value": 0
             },
+            "release_date": "2020-03-30T19:50:21.127849",
             "timeFrom": "now-2y",
             "timeRestore": true,
             "timeTo": "now",
-            "title": "AttributeTemplate",
+            "title": "activity_docker-smells",
```
Comment (vchrombie) [inline django-prosoul/prosoul/panels/templates/attribute-template.json]: same as the above comment, :thinking:

File: django-prosoul/prosoul/panels/templates/attribute-template.json [hunk 4]
```diff
@@ -1,23 +1,24 @@
...
             },
             "optionsJSON": "{\"darkTheme\":false}",
-            "panelsJSON": "[{\"col\":9,\"id\":\"AWG3EsMiETw-8I6Z4dX3\",\"panelIndex\":14,\"row\":1,\"size_x\":4,\"size_y\":4,\"type\":\"visualization\"},{\"col\":4,\"id\":\"AWG3ImxlETw-8I6Z4dX5\",\"panelIndex\":15,\"row\":1,\"size_x\":5,\"size_y\":4,\"type\":\"visualization\"},{\"col\":1,\"id\":\"AWG3IxdRETw-8I6Z4dX6\",\"panelIndex\":16,\"row\":1,\"size_x\":3,\"size_y\":4,\"type\":\"visualization\"}]",
+            "panelsJSON": "[{\"col\": 9, \"id\": \"AWG3EsMiETw-8I6Z4dX3\", \"panelIndex\": 14, \"row\": 1, \"size_x\": 4, \"size_y\": 4, \"type\": \"visualization\"}, {\"col\": 4, \"id\": \"AWG3ImxlETw-8I6Z4dX5\", \"panelIndex\": 15, \"row\": 1, \"size_x\": 5, \"size_y\": 4, \"type\": \"visualization\"}, {\"col\": 1, \"id\": \"AWG3IxdRETw-8I6Z4dX6\", \"panelIndex\": 16, \"row\": 1, \"size_x\": 3, \"size_y\": 4, \"type\": \"visualization\"}]",
```
Comment (vchrombie) [inline django-prosoul/prosoul/panels/templates/attribute-template.json]: same as the above comment, :thinking:

File: django-prosoul/prosoul/panels/templates/attribute-template.json [hunk 5]
```diff
@@ -1,23 +1,24 @@
...
+                "searchSourceJSON": "{\"filter\": [{\"$state\": {\"store\": \"appState\"}, \"meta\": {\"alias\": null, \"disabled\": false, \"index\": \"ossmeter\", \"key\": \"metric_name\", \"negate\": false, \"params\": [\"Sum of Docker Smells\", \"Sum of Docker Dependencies\"], \"type\": \"phrases\", \"value\": \"Sum of Docker Smells, Sum of Docker Dependencies\"}, \"query\": {\"bool\": {\"minimum_should_match\": 1, \"should\": [{\"match_phrase\": {\"metric_name\": \"Sum of Docker Smells\"}}, {\"match_phrase\": {\"metric_name\": \"Sum of Docker Dependencies\"}}]}}}, {\"query\": {\"match_all\": {}}}], \"highlightAll\": true, \"version\": true}"
```
Comment (vchrombie) [inline django-prosoul/prosoul/panels/templates/attribute-template.json]: same as the above comment, :thinking:

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 1 above)
Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: I understand that this should be reverted, since the template is generic. Can you check that the new ID value isn't used anywhere? thanks

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 2 above)
Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: I understand it shouldn't be reverted

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 3 above)
Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: Yes, I would revert it

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 4 above)
Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: This doesn't need to be reverted

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 1 above)
Comment (vchrombie) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: Sure, it was replaced one more time. I would revert these changes.
Thanks.

File: django-prosoul/prosoul/panels/templates/attribute-template.json (hunk 5 above)
Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: That's strange, the new index value is an old one. It should be `ea1313b0-8f29-11e8-a362-0d37aacd7dee`

I'm wondering if the attribute-template.json is used.

Comment (vchrombie) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: Oh, anything wrong with this change? :thinking:

Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: It seems so.

Can you try to make a test with the previous version and see what happens? thanks

Comment (vchrombie) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: Hi @valeriocos 
I reverted only this change (removed this line and added the previous line) and I performed a visualization again (deleted the dashboard too before viz) and it worked. The dashboard was generated normally.

I think we can revert this change, WDYT?

Comment (valeriocos) [reply django-prosoul/prosoul/panels/templates/attribute-template.json]: perfect, thanks @vchrombie!

**Comment by vchrombie** [comment]
Done! :tada:
